packages/node/test/transports/new/http.test.ts

```diff
@@ -3,7 +3,7 @@ import { EventEnvelope, EventItem } from '@sentry/types';
 import { createEnvelope, serializeEnvelope } from '@sentry/utils';
 import * as http from 'http';
 
-import { makeNewHttpTransport } from '../../../src/transports/new';
```

Comment:
We’ll need to rename this file as well

Reply:
Thanks! c1f5fa65d355f57129abc7c43a0fb60d5b0865b4